setup.cfg

```diff
@@ -53,6 +53,8 @@ tests =
     simplejson>=3.8
     sphinx>=4.5
     invenio-db[mysql,postgresql,versioning]>=1.0.9,<2.0.0
+    flask-breadcrumbs
```

Comment:
Minor: This should be removed, looks like we have dropped support for flask-breadcrumbs in invenio-theme (here the [commit](https://github.com/inveniosoftware/invenio-theme/commit/1cec8367f2b96dcfee3341a18630cee805ca03be)) and it will alos be removed from all the repositories soon (this afternoon most likely)
```suggestion
```